compiler/damlc/daml-doc/src/DA/Daml/Doc/HaddockParse.hs

```diff
@@ -136,15 +148,33 @@ data DeclData = DeclData
 
 buildDocCtx :: TypecheckedModule -> DocCtx
 buildDocCtx dc_tcmod  =
-  let dc_mod
-          = Modulename . T.pack . moduleNameString . moduleName
-          . ms_mod . pm_mod_summary . tm_parsed_module $ dc_tcmod
+  let dc_mod = packModule . ms_mod . pm_mod_summary . tm_parsed_module $ dc_tcmod
```

Comment:
```suggestion
  let dc_modname = packModule . ms_mod . pm_mod_summary . tm_parsed_module $ dc_tcmod
```
Makes it a bit clearer what is going on. `packModule` also seems like a weird name for getting the module name from a module.